README.md

```diff
@@ -1,22 +1,74 @@
-# RSpec Scaffolding [![Build Status](https://travis-ci.org/ridiculous/rspec-scaffold.svg)](https://travis-ci.org/ridiculous/rspec-scaffold)
+# RSpec Scaffold [![Build Status](https://travis-ci.org/ridiculous/rspec-scaffold.svg)](https://travis-ci.org/ridiculous/rspec-scaffold)
 
-Generates RSpec scaffolding for existing code. Helps you write tests by showing you what you should be testing, 
-which are conditions and changes to state (at a minimum).
+Generates RSpec scaffolding for existing code.  
+Cleverly infers spec location from source file location.   
+Helps you write tests by showing you what you should be testing, which are conditions and changes to state (at a minimum).
 
 ## Installation
 
 Requires Ruby >= 1.9.3
 
+__1. update gemfile and bundle__
+
 ```ruby
-gem 'rspec-scaffold'
+gem 'rspec-scaffold', '~> 2.0.0.beta1', require: false
+```
+
+__2. see if it works__
+
+```
+rspec-scaffold
 ```
 
+## Caveats
+Works best if used from within a Rails app root and if the project has the spec/ directory (it should!).  
+
 ## Usage
+The gem provides a command line utility `rspec-scaffold` for working with existing files, and simple-to-use module methods for programmatic use.  
+The idea is to point to existing ruby code files and `rspec-scaffold` will ensure corresponding spec files in `/spec` directory.  
+
+### The CLI
+Only operates on paths.
```

Comment:
It is supposed to mean that CLI only takes files/paths as arguments, not raw ruby code.